### PR TITLE
chore: change e.printStackTrace to logging

### PR DESCRIPTION
**Pull request by IgnacioDomingo**
Change several classes to remove calls to printStackTrace of exceptions
and use instead of logging from SFL4J which is already used in the
project

Ammendment of previous commit so the format and whitespacing is the same

### CODE REVIEW

File: src/main/java/com/rometools/modules/base/GoogleBaseImpl.java [hunk 1]
```diff
@@ -68,6 +71,8 @@
      *
      */
     private static final long serialVersionUID = 1L;
+    
+    private static final Logger		LOG	= LoggerFactory.getLogger(GoogleBaseImpl.class);
```
Comment (PatrickGotthard) [inline src/main/java/com/rometools/modules/base/GoogleBaseImpl.java]: Please correct intendation

File: src/main/java/com/rometools/modules/base/io/GoogleBaseGenerator.java [hunk 2]
```diff
@@ -72,6 +74,8 @@
  */
 public class GoogleBaseGenerator implements ModuleGenerator {
     private static final Namespace NS = Namespace.getNamespace("g-core", GoogleBase.URI);
+    
+    private static final Logger		LOG	= LoggerFactory.getLogger(GoogleBaseGenerator.class);
```
Comment (PatrickGotthard) [inline src/main/java/com/rometools/modules/base/io/GoogleBaseGenerator.java]: Please correct intendation

File: src/main/java/com/rometools/modules/cc/types/License.java [hunk 3]
```diff
@@ -45,6 +45,10 @@
 import java.util.StringTokenizer;
 import java.util.concurrent.ConcurrentHashMap;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.rometools.modules.base.GoogleBaseImpl;
```
Comment (PatrickGotthard) [inline src/main/java/com/rometools/modules/cc/types/License.java]: Remove import

File: src/main/java/com/rometools/modules/cc/types/License.java [hunk 4]
```diff
@@ -56,6 +60,7 @@
 
     private static final String CC_START = "http://creativecommons.org/licenses/";
     private static final Map<String, License> lookupLicense = new ConcurrentHashMap<String, License>();
+    private static final Logger		LOG	= LoggerFactory.getLogger(License.class);
```
Comment (PatrickGotthard) [inline src/main/java/com/rometools/modules/cc/types/License.java]: Correct intendation

File: src/main/java/com/rometools/modules/georss/GeoRSSModule.java [hunk 5]
```diff
@@ -17,7 +17,10 @@
 package com.rometools.modules.georss;
 
 import org.jdom2.Namespace;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import com.rometools.modules.base.GoogleBaseImpl;
```
Comment (PatrickGotthard) [inline src/main/java/com/rometools/modules/georss/GeoRSSModule.java]: Remove import

File: src/main/java/com/rometools/modules/georss/GeoRSSModule.java [hunk 6]
```diff
@@ -34,6 +37,8 @@
 public abstract class GeoRSSModule extends ModuleImpl implements Cloneable {
 
     private static final long serialVersionUID = 1L;
+    
+    private static final Logger		LOG	= LoggerFactory.getLogger(GeoRSSModule.class);
```
Comment (PatrickGotthard) [inline src/main/java/com/rometools/modules/georss/GeoRSSModule.java]: Correct intendation

File: src/main/java/com/rometools/modules/opensearch/impl/OpenSearchModuleImpl.java [hunk 7]
```diff
@@ -18,6 +18,10 @@
 import java.util.LinkedList;
 import java.util.List;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.rometools.modules.base.GoogleBaseImpl;
```
Comment (PatrickGotthard) [inline src/main/java/com/rometools/modules/opensearch/impl/OpenSearchModuleImpl.java]: Remove import

File: src/main/java/com/rometools/modules/opensearch/impl/OpenSearchModuleImpl.java [hunk 8]
```diff
@@ -29,6 +33,7 @@
  */
 public class OpenSearchModuleImpl extends ModuleImpl implements OpenSearchModule, Serializable {
     private static final long serialVersionUID = 1L;
+    private static final Logger		LOG	= LoggerFactory.getLogger(OpenSearchModuleImpl.class);
```
Comment (PatrickGotthard) [inline src/main/java/com/rometools/modules/opensearch/impl/OpenSearchModuleImpl.java]: Correct intendation

File: src/main/java/com/rometools/modules/sse/modules/Conflict.java [hunk 9]
```diff
@@ -17,6 +17,10 @@
 
 import java.util.Date;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.rometools.modules.base.GoogleBaseImpl;
```
Comment (PatrickGotthard) [inline src/main/java/com/rometools/modules/sse/modules/Conflict.java]: Remove import

File: src/main/java/com/rometools/modules/sse/modules/Conflict.java [hunk 10]
```diff
@@ -25,6 +29,8 @@
  */
 public class Conflict extends SSEModule {
     private static final long serialVersionUID = 1L;
+    
+    private static final Logger		LOG	= LoggerFactory.getLogger(Conflict.class);
```
Comment (PatrickGotthard) [inline src/main/java/com/rometools/modules/sse/modules/Conflict.java]: Correct intendation

File: src/main/java/com/rometools/modules/sse/modules/SSEModule.java [hunk 11]
```diff
@@ -29,6 +31,8 @@
  */
 public abstract class SSEModule implements Module {
     private static final long serialVersionUID = 1L;
+    
+    private static final Logger		LOG	= LoggerFactory.getLogger(SSEModule.class);
```
Comment (PatrickGotthard) [inline src/main/java/com/rometools/modules/sse/modules/SSEModule.java]: Correct intendation

**Comment by PatrickGotthard** [comment]
Hi ignaciodomingo,

thanks for your pull request. I'll merge it when you fix the intendation and imports.

Regards,
Patrick

**Comment by ignaciodomingo** [comment]
I have made a new commit fixing those things (sorry, had to deactivate the automation tools of eclipse so i didn't change your current format and forgot to do these checks :P)

File: src/main/java/com/rometools/modules/base/GoogleBaseImpl.java [hunk 12]
```diff
@@ -68,6 +71,8 @@
      *
      */
     private static final long serialVersionUID = 1L;
+    
+    private static final Logger	LOG	= LoggerFactory.getLogger(GoogleBaseImpl.class);
```
Comment (mishako) [inline src/main/java/com/rometools/modules/base/GoogleBaseImpl.java]: Replace tabs with spaces:
```
private static final Logger LOG = LoggerFactory.getLogger(GoogleBaseImpl.class);
```

File: src/main/java/com/rometools/modules/base/GoogleBaseImpl.java [hunk 13]
```diff
@@ -1297,7 +1302,7 @@ public Object clone() throws CloneNotSupportedException {
 
             return gbi;
         } catch (final Exception e) {
-            e.printStackTrace();
+            LOG.error("Error",e);
```
Comment (mishako) [inline src/main/java/com/rometools/modules/base/GoogleBaseImpl.java]: Space after comma: `LOG.error("Error", e);`

File: src/main/java/com/rometools/modules/base/io/GoogleBaseGenerator.java [hunk 14]
```diff
@@ -72,6 +74,8 @@
  */
 public class GoogleBaseGenerator implements ModuleGenerator {
     private static final Namespace NS = Namespace.getNamespace("g-core", GoogleBase.URI);
+    
+    private static final Logger	LOG	= LoggerFactory.getLogger(GoogleBaseGenerator.class);
```
Comment (mishako) [inline src/main/java/com/rometools/modules/base/io/GoogleBaseGenerator.java]: Tabs -> spaces.

File: src/main/java/com/rometools/modules/cc/types/License.java [hunk 15]
```diff
@@ -56,6 +59,7 @@
 
     private static final String CC_START = "http://creativecommons.org/licenses/";
     private static final Map<String, License> lookupLicense = new ConcurrentHashMap<String, License>();
+    private static final Logger	LOG	= LoggerFactory.getLogger(License.class);
```
Comment (mishako) [inline src/main/java/com/rometools/modules/cc/types/License.java]: Tabs -> spaces.

File: src/main/java/com/rometools/modules/cc/types/License.java [hunk 16]
```diff
@@ -120,7 +124,7 @@ public static License findByValue(final String uri) {
                         }
                     }
                 } catch (final Exception e) {
-                    e.printStackTrace();
+                	 LOG.error("Error",e);
```
Comment (mishako) [inline src/main/java/com/rometools/modules/cc/types/License.java]: Tabs -> spaces.

**Comment by mishako** [comment]
@IgnacioDomingo Sorry that I'm only commenting your code formatting, but it is really important part of software development. I haven't commented every improperly formatted line, because there are too many of them. Please go through and check every line in your change set.

**Comment by IgnacioDomingo** [comment]
@mishako I agree with you that code formatting is a very important thing, but we should strive to do this automatically. Do you have any formatting template for eclipse or another IDE that I can use to make sure the formatting is exactly the same one needed?

**Comment by mishako** [comment]
@IgnacioDomingo I'm not sure if Rome has any code formatting conventions. But, judging from the code I've seen, I would assume that Google style [1] would fit if you change indentation from 2 to 4 and max line length from 100 to 200.
[1] https://github.com/google/styleguide/blob/gh-pages/eclipse-java-google-style.xml

**Comment by PatrickGotthard** [comment]
Hi together,

we really should place the code conventions somewhere. Just to solve this discussion: 4 spaces are used for intendation and a line should not exceed 160 chars (including whitespace).

Regards,
Patrick

**Comment by IgnacioDomingo** [comment]
Ok, then I'm going to use the Google Style template with indentation of 4 spaces and max line size of 160 chars. OK?

**Comment by PatrickGotthard** [comment]
That should be okay for now. When I find time I will add a CheckStyle configuration that is independent from any IDE.

**Comment by mishako** [comment]
:+1:

**Comment by IgnacioDomingo** [comment]
:)

**Pull request merged**